jwst/assign_wcs/tests/test_nirspec.py

```diff
@@ -57,7 +57,7 @@ def create_hdul():
     phdu.header['instrume'] = 'NIRSPEC'
     phdu.header['detector'] = 'NRS1'
     phdu.header['time-obs'] = '8:59:37'
-    phdu.header['date-obs'] = '2014-09-05'
+    phdu.header['date-obs'] = '2016-09-05'
```

Comment:
Regardless, I think the `DATE-OBS` here should be later, as even when pmaps ≥ 349 become default, the earlier files will be selected, as the later newer contexts will have USEAFTER dates for the CV3 data, i.e. ~Nov 2015.